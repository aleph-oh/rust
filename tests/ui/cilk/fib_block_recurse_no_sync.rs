fn fib(n: usize) -> usize {
    if n <= 1 {
        return n;
    }
    let x = cilk_spawn { fib(n - 1) };
//~^ ERROR used binding `x` is possibly-uninitialized [E0381]
    let y = fib(n - 2);
    x + y
}

fn main() {

}
