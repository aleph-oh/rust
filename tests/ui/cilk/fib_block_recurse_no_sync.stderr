error[E0381]: used binding `x` is possibly-uninitialized
  --> $DIR/fib_block_recurse_no_sync.rs:8:9
   |
LL |     let x = cilk_spawn { fib(n - 1) };
   |         ^
   |         |
   |         `x` used here but it is possibly-uninitialized
   |         binding declared here but left uninitialized

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0381`.
